scipy/fftpack/basic.py

```diff
@@ -378,6 +378,19 @@ def fft2(x, shape=None, axes=(-2,-1), overwrite_x=False):
     --------
     fftn : for detailed information.
 
+    Examples
+    --------
+    >>> from scipy.fftpack import fft2, ifft2
+    >>> import numpy as np
```

Comment:
Could you remove this import, as well as the `numpy` ones in the other examples? These imports are assumed for every function in SciPy, to avoid repeating it 1000+ times over.